sentinel-dashboard/src/main/resources/application.properties

```diff
@@ -12,6 +12,7 @@ logging.pattern.file= %d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} - %
 #auth settings
 auth.filter.exclude-urls=/,/auth/login,/auth/logout,/registry/machine,/version
 auth.filter.exclude-url-suffixes=htm,html,js,css,map,ico,ttf,woff,png
+auth.enabled=false
```

Comment:
How about making the default value true? since for most users, they may need login validation.

Reply:
Yes, I have modified